### PR TITLE
Fix #472, Section1 Introduction - Clarified the meaning of mixed audio element

### LINKED ISSUE #472: Section 1 Introduction Improvement
- Need to clarify the meaning of mixed audio element
- Typo correction
- etc.

### PR DESCRIPTION
**Pull request by sunghee-hwang**
with some typos fixing

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/480.html" title="Last updated on Jun 27, 2023, 11:51 PM UTC (0eec9a5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/480/2928138...0eec9a5.html" title="Last updated on Jun 27, 2023, 11:51 PM UTC (0eec9a5)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -268,13 +268,14 @@ The model includes one or more [=Audio Element=]s, each of which consists of one
...
 	- [=Parameter Substream=] may change their values over time and may further be animated; for example, any changes in values may be smoothed over some time duration. As such, they may be viewed as a 1D signal that have different metadata specified for different time durations.
-- The term <dfn noexport>Mix Presentation</dfn> means a series of processes to present [=Immersive Audio=] contents to end-users by using [=Audio Element=](s). It contains metadata, that describe how the [=Audio Element=]s are rendered and mixed together for playback through physical loudspeakers or headsets, and loudness information.
+- The term <dfn noexport>Mix Presentation</dfn> means a series of processes to present [=Immersive Audio=] contents to end-users by using [=Audio Element=](s). It contains metadata, that describe how the [=Audio Element=](s) is(are) rendered and mixed together for playback through physical loudspeakers or headsets, and loudness information.
+- The term <dfn noexport>Mixed Audio Element</dfn> means a [=3D audio signal=] after the [=Audio Element=](s) defined in a [=Mix Presenation=] is(are) rendered and mixed together for playback through physical loudspeaker or headsets.
```
Comment (tdaede) [inline index.bs:272]: Mixed Audio Element -> Rendered Mix Presentation

**Comment by sunghee-hwang** [comment]
@felicialim, I would suggest to update figure 1 to address #489 in this PR.

File: index.bs (hunk 1 above)
Comment (sunghee-hwang) [reply index.bs]: addressed

File: index.bs [hunk 2]
```diff
@@ -2660,7 +2667,7 @@ For a given channel-based input audio (e.g. 7.1.4ch) and a given list of channel
 		- Effect scene: Directional and spatially moving sounds.
 	- Height Energy Quantification module generates a surround to height mixing parameter (w(k)) which is decided according to the relative energy difference between the top and surround channels of the input audio.
 		- If the energy of top channels is bigger than that of surround ones, then w_idx_offset(k) is set to 1. Otherwise, it is set to -1. And, w(k) is calculated based on w_idx_offset(k) and conforms to [[#processing-scalablechannelaudio]].
-- Down-mixer generates down-mixed audios from the input audio according to the list of channel layouts and the down-mix parameters, and outputs [=down-mixed audio=] for each channel layout to Loudness module.
+- Down-mixer generates [=down-mixed audio=]s from the input audio according to the list of channel layouts and the down-mix parameters, and outputs [=down-mixed audio=] for each channel layout to Loudness module.
```
Comment (tdaede) [inline index.bs:2670]: don't need the s

**Review by tdaede — APPROVED** [review]

File: index.bs (hunk 2 above)
Comment (sunghee-hwang) [inline index.bs:2670]:
```suggestion
- Down-mixer generates [=down-mixed audio=] from the input audio according to the list of channel layouts and the down-mix parameters, and outputs [=down-mixed audio=] for each channel layout to Loudness module.
```

**Review by sunghee-hwang — COMMENTED** [review]
delete 's'

File: index.bs (hunk 2 above)
Comment (sunghee-hwang) [reply index.bs]: done

File: index.bs [hunk 3]
```diff
@@ -2539,9 +2546,9 @@ All of syntax elements conform to [=Syntactic Description Language=] specified i
  
  <b>string</b> <b>syntaxName</b>
 
-<b>string</b> indicates a null-terminated (i.e. ending at the first byte set to 0x00), UTF-8 encoded as defined in [[!RFC3629]] and whose length shall be limited to 128 bytes. 
+<b>string</b> indicates a null-terminated (i.e. ending at the first byte set to 0x00), UTF-8 encoded as defined in [[!RFC3629]] and whose length SHALL be limited to 128 bytes and SHALL not contain the UTF-8 NULL character (0x80) except for the null termination.
```
Comment (sunghee-hwang) [inline index.bs:2549]:
```suggestion
<b>string</b> indicates a null-terminated (i.e. ending at the first byte set to 0x00), UTF-8 encoded as defined in [[!RFC3629]] and whose length shall be limited to 128 bytes.
```

File: index.bs [hunk 4]
```diff
@@ -2539,9 +2546,9 @@ All of syntax elements conform to [=Syntactic Description Language=] specified i
  
  <b>string</b> <b>syntaxName</b>
 
-<b>string</b> indicates a null-terminated (i.e. ending at the first byte set to 0x00), UTF-8 encoded as defined in [[!RFC3629]] and whose length shall be limited to 128 bytes. 
+<b>string</b> indicates a null-terminated (i.e. ending at the first byte set to 0x00), UTF-8 encoded as defined in [[!RFC3629]] and whose length SHALL be limited to 128 bytes and SHALL not contain the UTF-8 NULL character (0x80) except for the null termination. 
  
-<b>syntaxName</b> is a human readable label. The label shall not include the number 0.
+<b>syntaxName</b> is a human readable label.
```
Comment (sunghee-hwang) [inline index.bs:2551]:
```suggestion
<b>syntaxName</b> is a human readable label.  The label shall not include the number 0.
```

**Review by sunghee-hwang — COMMENTED** [review]
It is reverted to be address by a new PR

**Pull request merged**